Makefile

```diff
@@ -460,21 +475,21 @@ XRUN_DISABLED_WARNINGS 	:= $(patsubst %, -nowarn %, $(XRUN_DISABLED_WARNINGS))
 XRUN_COMP = $(XRUN_COMP_FLAGS)		\
 	$(XRUN_DISABLED_WARNINGS) 	\
 	$(XRUN_INCDIR)		      	\
+	-f ../core/Flist.cva6    	\
 	$(filter %.sv, $(ariane_pkg)) 	\
-	$(filter %.vhd, $(uart_src))  	\
 	$(filter %.sv, $(src))	      	\
-	-f ../core/Flist.cva6    	    \
-	$(filter %.sv, $(XRUN_TB))	\
+	$(filter %.vhd, $(uart_src))  	\
```

Comment:
Xcelium is not able to compile **ariane_pkg** first, since some dependencies cannot be fulfilled at that time with the files given in **$(ariane_pkg)**. Anyway, if that swap is not acceptable still it'll be necessary to update **$(ariane_pkg)**, to especially give the missing dependencies or to swap something else... I'll take a look what can be done